src/libs/actions/App.js

```diff
@@ -124,20 +124,35 @@ function openApp() {
                         key: ONYXKEYS.IS_LOADING_REPORT_DATA,
                         value: true,
                     },
+                    {
+                        onyxMethod: CONST.ONYX.METHOD.MERGE,
+                        key: ONYXKEYS.IS_LOADING_INITIAL_APP_DATA,
```

Comment:
This was removed in https://github.com/Expensify/App/pull/12722 but I have added it back. We can also use `IS_LOADING_REPORT_DATA` let me know if I should use `IS_LOADING_REPORT_DATA` instead.